studio/data/permissions/permissions-query.ts

```diff
@@ -27,7 +27,7 @@ export const usePermissionsQuery = <TData = PermissionsData>(
   useQuery<PermissionsData, PermissionsError, TData>(
     permissionKeys.list(),
     ({ signal }) => getPermissions(signal),
-    options
+    { enabled: IS_PLATFORM && options.enabled, ...options }
```

Comment:
`options.enabled` actually needs to be destructured from `options`, otherwise `...options` will just override it